dev-docs/bidders/valueimpression.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: Valueimpression
+description: Prebid Valueimpression Bidder Adapter
+hide: true
+biddercode: valueimpression
+media_types: banner, video
+gdpr_supported: true
+schain_supported: true
+usp_supported: true
+---
+
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name        | Scope    | Description                                                                                                                    | Example    | Type     |
+|-------------|----------|--------------------------------------------------------------------------------------------------------------------------------|------------|----------|
+| `siteId`    | required | Publisher  GUID from 33Across                                                                                                  | `'vi-site-id'` | `string` |
```

Comment:
What does 33across have to do with your bidder? Is this a copy/paste error?

Reply:
Oh, sorry. I got a little mistaken times while copying